Framework/Core/Windows/Devices/I2c/I2cEnums.cs

```diff
@@ -0,0 +1,21 @@
+namespace Windows.Devices.I2c
+{
+    public enum I2cBusSpeed
+    {
+        StandardMode = 0,   // 100kHz

```

Comment:
It is a good idea to be explicit with the 0 value as enums are value types and the default integral value is 0, thus the result of default( T ) where T is an enum is well known and explicitly declared.